src/controllers/expense.controllers.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const expenseServices = require('../services/expense.service');
+const userServices = require('../services/user.service');
```

Comment:
```suggestion
const expensesServices = require('../services/expense.service');
const usersServices = require('../services/user.service');
```

The same in all other places